core/documentation/development/tooling/TestingLocalChanges.mdx

```diff
@@ -22,9 +25,7 @@ In some projects you might have to do some additional configuration to enable sy
 3. Then navigate to your project where you want to link the Fudis core package. The directory that contains the node_modules folder containing the project dependencies is the correct place to execute linking. In this project directory run `npm link @funidata/fudis-core`
 4. When you want to unlink the package from your project, run the command `npm unlink @funidata/fudis-core`. This is done in the same directory in your consuming project, where you linked the package.
 
-If you want to remove the global symlink, go back to the **core/dist** folder and run `npm unlink`.
```

Comment:
I think this should be the 5th step and not deleted, otherwise the linking will hunt you next time you try to use the symlink.

Reply:
Same comment as in Ngx, causes an error.

Reply:
Maybe this would need `npm unlink -g` (reference https://stackoverflow.com/questions/69906798/unable-to-remove-symlinks-after-running-npm-link)